app.py

```diff
@@ -1,33 +1,85 @@
-from flask import Flask
+from flask import Flask, g, json, jsonify, render_template, redirect, request, url_for
 from flask_cors import CORS
 from werkzeug import exceptions
 import sqlite3
+from controllers import urls
+
 
 app = Flask(__name__)
 CORS(app)
 
 DATABASE = 'database/database.db'
+URL = 'shoter.io'
 
-@app.route("/")
-def homepage(): 
-    init_db()
-    return "Welcome!"
+@app.route("/", methods=['GET','POST'])
+def homepage():
+  if request.method == 'GET': 
+    return render_template('home.html', title="Save URL")
+  elif request.method == 'POST':
+    response, code = urls.create(request)
+    if code == 201:
+      content = f'A new URL:  {request.form["url"]}, has been saved as '
+      link = f'http://127.0.0.1:5000/aliases/{response[0][1]}'
+      link_name =f'{URL}/{response[0][1]}'
+      return render_template('home.html', title="SAVED", content=content, link=link, linkName=link_name)
+    else: 
+      return redirect(url_for('aliases', alias=response))
```

Comment:
I couldn't seem to get the short.io link to redirect to original link. So https://github.com/roselynle returned back a short link of short.io/mxhkc. However when I visit short.io/mxhkc it returns the 404
error page (on the official short.io page). I think the redirect method needs tweaking to redirect the user to the original link saved on the database when they visit the short link?